blog/2025/pcsx2-2.6/index.mdx

```diff
@@ -0,0 +1,284 @@
+---
+draft: false
+title: PCSX2 2.6.0 is now out!
+tags:
+  - devblog
+date: 2025-12-31
+description: PCSX2 2.6 Release Post
+hide_table_of_contents: false
+titleImage: /img/2-0-banner.webp
+image: /img/2-0-banner.webp
+authors:
+  - kamfretoz
+  - govanify
+  - pandubz
+  - jordan
+  - link
+---
+
+Nearly 6 months after PCSX2 2.4.0 release, we are now nearing the end of 2025.
+
+**Today we are excited to unveil PCSX2 2.6.0!** We're even more excited to showcase the things that we have been working on.
+
+<!--truncate-->
+
+## PCSX2 2.6.0
+
+Now that the dust has settled down from [our biggest single release to date](../../2024/pcsx2-2-release/index.mdx), we decided to step back a bit and give more attention to Quality of Life features and Accessibility! Our contributors have been hard at work further refining the foundation that has been laid out since 2.0.0 to make PCSX2 more accessible to everyone!
+
+<!--PR List-->
+
+## Major Spotlight
+
+<PCSX2PRLink authors="SternXD" prNums="12740,12944,13294">
+  ### Better Feature Parity Between Big Picture Mode and Qt Interface
+</PCSX2PRLink>
+
+SternXD has been on a mission to achieve feature parity between Big Picture Mode and the Qt interface. Thanks to his work, you can now adjust Network and HDD Settings, Create a Memory Card as well as login to RetroAchievements directly from Big Picture Mode!
+
+<Image src={require("./img/BPM_network.webp").default} />
+
+<PCSX2PRLink authors="TheTechnician27" prNums="13199">
+  ### Korean Game Titles
+</PCSX2PRLink>
+
+TheTechnician27 has been working on adding Korean game titles to most NTSC-K serials which did not previously have them. PCSX2 now has a 99% complete Korean translation on Crowdin (at the time of writing), so this has been a long time coming. Despite NTSC-K having nearly 700 known serials, the GameDB for a long time has only had Latin versions of the titles, either pulled from the NTSC-U/PAL name or from a Romaji version of the Japanese name.
+
+<PCSX2PRLink authors="kamfretoz" prNums="12746">
+  ### Game Shortcut Creation Feature
+</PCSX2PRLink>
+
+Motivated by [this GitHub issue](https://github.com/PCSX2/pcsx2/issues/8349), KamFretoZ has set his sights on implementing a feature where you can automatically create a desktop (or start menu) shortcut for your game right from PCSX2 so you can launch your games right from your desktop! This one has been requested a lot and we've seen quite a few people ask about it on our Discord server.
+
+Only available for Windows and Linux.
```

Comment:
Just to make it a little more clear, "This feature is only available for Windows and Linux at this time."